### PR TITLE
refactor(yaml): Update openebs-base with the latest changes

### PR DESCRIPTION
**Pull request by w3aman**
1. Added "cspc-stripe-pool job" in set up stage
2. Added jobs in clean up stage.
3. update resource version in e2e-cr util.
4. modified util to dump the cluster state.
5. Added above updates in .gitlab-ci.yml file.

### CODE REVIEW

File: Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/1CXH-storage-policies/openebs-sc [hunk 1]
```diff
@@ -67,4 +67,4 @@ if [ "$1" == "node" ];then
   node $2 $3 $4
 else
   pod
-fi
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/1CXH-storage-policies/openebs-sc]: @w3aman there is no change in this file ?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -133,51 +141,51 @@ Q2JG-NFSv3-PROVISION:
...
+
 .func_test_template:
   image: openebs/openshift-ci:latest
   stage: FUNCTIONAL
   when: always
   dependencies:
-    - K9YC-OpenEBS-Deployment
+    - K9YC-OpenEBS
      
-9AKX-cStor-Create-Snapshot:
```
[the remark this answers is not in the document]
Comment (w3aman) [reply .gitlab-ci.yml]: Added back !! @nsathyaseelan

File: Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/1CXH-storage-policies/openebs-sc (hunk 1 above)
Comment (w3aman) [reply Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/1CXH-storage-policies/openebs-sc]: yes there is no change.

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -50,12 +51,19 @@ G45J-MIRRORED-POOL:
...
 
-9ZWW-RAIDZ2-POOL:
+# 9ZWW-RAIDZ2-POOL:
+#   image: openebs/openshift-ci:latest
+#   stage: OPENEBS-SETUP
+#   script:
+#     - chmod 775 ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/9ZWW-create-raidz2-pool/create-raidz2-pool
+#     - ./Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/9ZWW-create-raidz2-pool/create-raidz2-pool
+
+KUSG-CSPC_STRIPE_POOL:
```
Comment (nsathyaseelan) [inline .gitlab-ci.yml]: @w3aman can you change the job name into `KUSG-CSPC-STRIPE-POOL`

Comment (w3aman) [reply .gitlab-ci.yml]: Done !

**Pull request merged**